pkg/koordlet/metricsadvisor/collector.go

```diff
@@ -263,33 +276,36 @@ func (c *collector) collectPodResUsed() {
 			klog.Errorf("insert pod %s/%s, uid %s resource metric failed, metric %v, err %v",
 				pod.Namespace, pod.Name, uid, podMetric, err)
 		}
-		c.collectContainerResUsed(meta)
+
 	}
 
 	// update collect time
 	c.state.RefreshTime(podResUsedUpdateTime)
 	klog.Infof("collectPodResUsed finished, pod num %d", len(podMetas))
 }
 
-func (c *collector) collectContainerResUsed(meta *statesinformer.PodMeta) {
+func (c *collector) collectContainerResUsed(meta *statesinformer.PodMeta) (totalGPUUsages []*metriccache.GPUMetric) {
```

Comment:
why return `[]*metriccache.GPUMetric` here?

Reply:
get pod gpu usage from containers in pod. return array in order to support for multiple device.